.github/workflows/gateway-conformance.yml

```diff
@@ -60,6 +60,8 @@ jobs:
         run: |
           ./ipfs init
           ./ipfs config --json Gateway.PublicGateways "$GATEWAY_PUBLIC_GATEWAYS"
+          ./ipfs config --json Experimental.Libp2pStreamMounting true
+          ./ipfs config --json Experimental.Experimental.GatewayOverLibp2p true
```

Comment:
`Experimental.Experimental` ?
Also we are turning that experimental, I thought on by default ?

Reply:
Typo, which is fixed. However, yes we're leaving as experimental for the time being given that the API in go-libp2p is itself experimental (and the spec PRs there haven't landed yet) and it's new.

Hopefully we'll be able to promote this out of experimental mode in the near future once we gain some confidence in its use and figure out what kinds of knobs people want exposed in the config for it.